src/mlir/dialect/VeronaDialect.h

```diff
@@ -3,11 +3,8 @@
 
 #pragma once
 
-#include "dialect/VeronaTypes.h"
 #include "mlir/IR/Dialect.h"
 
-namespace mlir::verona
-{
+// This line must come after "normal" includes.
```

Comment:
Keeping it inside the namespace as before would have prevented that, too. Is the removal of the namespace a required change from the new tree?

Reply:
Sorry this is referenced in the commit message but not in the PR description.

Yes, all the namespace stuff is required by changes in MLIR. TableGen now wraps the generated code with `namespace` blocks, using whatever `cppNamespace` we define in the `.td`.

It’s possible we could leave `cppNamespace` empty, I’m not sure, but that would certainly not be the intended use case.

Cf. https://reviews.llvm.org/D86811

Reply:
Makes sense, thanks! Maybe we should list all the changes in each update and what made us do it, in the commit message.